pkg/visor/api.go

```diff
@@ -266,6 +267,11 @@ func (v *Visor) Summary() (*Summary, error) {
 		dmsgStatValue = &dmsgTracker
 	}
 
+	rewardAddress, err := v.GetRewardAddress()
+	if err != nil {
+		return nil, fmt.Errorf("reward_address")
+	}
```

Comment:
I think is better to export nil reward_address value instead return error on summary, isn't?